src/main/java/net/countercraft/movecraft/combat/features/tracking/BlockBehaviorOverride.java

```diff
@@ -0,0 +1,420 @@
+                }
+                if (igniteOddOverride >= 0) {
+                    putIgniteOddsFunction.accept(m, Pair.of(igniteOddOverride, igniteOddVanilla));
+                }
+            }
+        }
+    }
+
+    protected static void loadBlastResistanceValues(FileConfiguration config, Function<Material, Boolean> addToSet, BiConsumer<Material, Float> putFunction) {
+        if (!config.contains("BlastResistanceOverride"))
+            return;
+        var section = config.getConfigurationSection("BlastResistanceOverride");
+        if (section == null)
+            return;
+
+        for (var entry : section.getValues(false).entrySet()) {
+            EnumSet<Material> materials = Tags.parseMaterials(entry.getKey());
+            for (Material m : materials) {
+                float value;
+                String valStr = entry.getValue().toString();
+                try {
+                    value = Float.parseFloat(valStr);
+                } catch (NumberFormatException | NullPointerException ex) {
+                    MovecraftCombat.getInstance().getLogger()
+                            .warning("Unable to load " + m.name() + ": " + entry.getValue());
+                    continue;
+                }
+                addToSet.apply(m);
+                putFunction.accept(m, value);
+            }
+        }
+    }
+
+    public static void enable() {
+        processOverriddenEntry(BlockBehaviorOverride::set, "Failed to set block overrides!");
+    }
+
+    public static void disable() {
+        processOverriddenEntry(BlockBehaviorOverride::reset, "Failed to revert block overrides to vanilla!");
+    }
+
+    protected static void processOverriddenEntry(final BiFunction<Material, BlockOverride, Boolean> function, final String msgOnException) {
+        try {
+            for (Map.Entry<Material, BlockOverride> entry : BLOCK_OVERRIDES.entrySet()) {
+                if (!function.apply(entry.getKey(), entry.getValue()))
+                    MovecraftCombat.getInstance().getLogger().warning("Unable to set " + entry.getKey().name());
+            }
+        } catch (Exception e) {
+            MovecraftCombat.getInstance().getLogger().info(msgOnException);
+            e.printStackTrace();
+        }
+    }
+
+    protected static boolean set(Material mat, BlockOverride override) {
+        boolean result = true;
+
+        // Blast resistance
+        if (override.blastResistanceOverride().isPresent()) {
+            result = result && NMS_HELPER.setBlastResistance(mat, override.blastResistanceOverride().get().floatValue());
+        }
+        // Burn oddity
+        if (override.burnOddity().isPresent()) {
+            result = result && NMS_HELPER.setBurnOdds(mat, override.burnOddity().get().intValue());
+        }
+        // Ignite oddity
+        if (override.igniteOddity().isPresent()) {
+            result = result && NMS_HELPER.setIgniteOdds(mat, override.igniteOddity().get().intValue());
+        }
```

Comment:
You can use `result &= _` for these, and there's no need to unbox the values with `.floatValue()` or `.intValue()`.